source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +223,36 @@ export default class WindowManager {
       return
     }
 
+
+
+    this._mainWindow.on('show', () => {
+      if (process.platform === 'win32' || process.platform === 'linux') {
+        
```

Comment:
Fails `yarn lint` : ` 229:73  error    Block must not be padded by blank lines padded-blocks`
Fails `yarn lint` : ` 230:1   error    Trailing spaces not allowed no-trailing-spaces`